docs/book/shipments.rst

```diff
@@ -4,17 +4,75 @@
 Shipments
 =========
 
-...
+A **Shipment** is a representation of a shipping request for an Order. Sylius supports multiple shipments per one Order.
+
+How to create a Shipment programmatically?
+''''''''''''''''''''''''''''''''''''''''''
+
+In order to create a new Shipment - use a factory. Assign a shipping method, and an order - without that data a Shipment would be useless.
+Finally add your shipment to the repository.
+
+.. code-block:: php
+
+    $shipment = $this->container->get('sylius.factory.shipment')->createNew();
+
+    $shipment->setMethod($shippingMethod);
+    $shipment->setOrder($order);
+
+    $this->container->get('sylius.repository.shipment')->add($shipment);
 
 The Shipment State Machine
 --------------------------
 
-...
+A Shipment that is attached to an Order will have its own state machine with such states available:
+``cart``, ``ready``, ``cancelled``, ``partially_shipped``, ``shipped``.
+
+The allowed transitions between these states are:
+
+.. code-block:: yaml
+
+   transitions:
+      request_shipping:
+          from: [cart]
+          to: ready
+      cancel:
```

Comment:
It is something for later, but I guess we are missing a transition between `partially_shipped` and `cancelled`